getting-started/setup-multi-node-auth.md

```diff
@@ -0,0 +1,351 @@
+# Authentication setup for multi-node
+
+Beyond the [basic setup][] for multi-node TimescaleDB, there are a number of 
+important aspects of the implementation that may need to be configured,
+especially for a production environment:
+- Enabling secure communication between the access node and the data 
+nodes using a superuser
+- Adding additional users/roles and permissions to interact with the system
+
+Setting up a secure system is a complex task and this section should not 
+be read as recommending any particular security measures for securing 
+your system.  That said, here are two technical examples for how to 
+enable authentication, [password authentication](#multi-node-auth-password) directly below and [certificate authentication](#multi-node-auth-certificate).
+
+
+### Prerequisites
+Both methods assume the existence of a superuser `postgres` with a preset password.
+
+---
+
+## Password authentication [](multi-node-auth-password)
+This method is for SCRAM SHA-256 password authentication. For other 
+password authentication methods, [see the PostgreSQL docs][auth-password].  Steps are as follows:
+
+A. _Each **data node** needs to be configured to accept and authenticate 
+connections using passwords._
+
+1. Set the correct password encryption method.
+2. Enable password authentication for the user.
+3. Reload the server to read the new configuration.
+4. Add the user on the data node with the right password.
+
+B. _The **access node** needs data node passwords for each role that will 
+use the access node._
+
+5. Set the correct password encryption method.
+6. Create a password file.
+7. Reload the server configuration.
+
+### A. Authentication on the data node
+Setting password authentication for superuser `postgres` using
+password `xyzzy`:
+
+##### 1. Set the password encryption method
+First edit the PostgreSQL configuration file `postgresql.conf` to set 
+the password encryption method to `scram-sha-256`.  Add this line to the 
+file:
+
+```
+password_encryption = 'scram-sha-256'		# md5 or scram-sha-256
+```
+
+Note that any previously created passwords will need to be recreated to 
+incorporate the new encryption.
+
+##### 2. Enable password authentication
+As with the [basic setup][], the instance authentication is set by modifying the 
+HBA file (by default `pg_hba.conf`, in the data directory). First, comment out the 
+HBA reference lin in `postgresql.conf`:
+
+```
+# Add a hash symbol to the start of the following line
+#hba_file = 'ConfigDir/pg_hba.conf' # host-based authentication file
+                                    # (change requires restart)
+```
```

Comment:
This is a direct copy from the previous version.  If it's not necessary, I can take it out

Reply:
Probably my fault. I think it can be confusing to have that `ConfigDir/pg_hba.conf` part of the file, so either remove it to use an example where the default is not used. (Probably easier to remove it.)